manager/apis/app/v1alpha1/blueprint_types.go

```diff
@@ -211,6 +211,7 @@ func CreateMetaBlueprintWithoutState(blueprint *Blueprint) MetaBlueprint {
 }
 
 const (
-	BlueprintNamespaceLabel = "app.fybrik.io/blueprintNamespace"
-	BlueprintNameLabel      = "app.fybrik.io/blueprintName"
+	BlueprintNamespaceLabel        = "app.fybrik.io/blueprintNamespace"
+	BlueprintNameLabel             = "app.fybrik.io/blueprintName"
```

Comment:
Why do we need these if it's always the system namespace?

Reply:
I take it back. The information might be useful for debugging. The `blueprintNamespace` name is misleading after this change tough.